kiss

```diff
@@ -1048,21 +1048,28 @@ pkg_remove_files() {
             }
         esac 2>/dev/null ||:
 
-        file=$KISS_ROOT/$file
+        _file=${KISS_ROOT:+"$KISS_ROOT/"}${file%%/}
```

Comment:
Can you explain why this has to happen? Doesn't this just do (for `KISS_ROOT=/test`)
`/test//path/to/file`
?

Reply:
If `KISS_ROOT` is empty this ends up being `//path/to/file`. This only adds the separator when it will be used. In addition, the trailing `/` is stripped from the path as this **breaks** the symlink tests below.

Reply:
I should have been more clear; is there a reason it has to be //? I know // resolves to / so it doesn't matter in that sense, so I'm confused about why it has to be done -- $file should already have a preceeding /, no?

Reply:
I see what you mean. ie, `$KISS_ROOT$file`. Will do some tests locally and fix.

Reply:
Fixed (and force pushed).